spec/ffi/fork_spec.rb

```diff
@@ -0,0 +1,41 @@
+#
+# This file is part of ruby-ffi.
+# For licensing, see LICENSE.SPECS
+#
+
+require File.expand_path(File.join(File.dirname(__FILE__), "spec_helper"))
+
+# Skip platforms that don't implement Process.fork
+unless RUBY_ENGINE == "truffleruby" or FFI::Platform.windows?
```

Comment:
A more reliable way to test that would be `Process.respond_to?(:fork)`
(https://github.com/oracle/truffleruby/blob/master/doc/user/compatibility.md#fork)
I'll push a commit to do that.

Reply:
522581c9d43544ec36d13a761ed43f5b6b891119

Reply:
Thanks, seems to work!